ext/debian/control

```diff
@@ -123,17 +123,15 @@ Description: Centralised configuration management - master setup to run under mo
 
 Package: vim-puppet
 Architecture: all
-Depends: ${misc:Depends}
+Depends: ${misc:Depends}, puppet (>= ${source:Version})
```

Comment:
Since vim is built as a subpackage of puppet in this paradigm, why not make the requirement = rather than >= ?